model/pod/pod.go

```diff
@@ -57,6 +57,11 @@ const (
 	StatusTerminated Status = "terminated"
 )
 
+var ActiveStatus = []Status{
```

Comment:
Add doc comment.

Reply:
nit: doc comments generally don't need to talk about type information like saying it's a slice of statuses, since that's type information that's readily available just by using a typed language. Since you haven't defined what "active" means, I would phrase it as something more along the lines of "ActiveStatus represents all statuses in which the pod exists in the cloud provider."

Alternatively, `ActiveStatus` could just be removed since it seems like it's not being used.

Reply:
I think we might end up using something like this in the future but will delete for now.